lib/http.js

```diff
@@ -47,8 +54,8 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   // register a new keystore
   app.post(
     routes.keystores,
+    validate({bodySchema: postKeystoreBody}),
     createMiddleware({strategy: 'session'}),
```

Comment:
Validation should come after session checks so as to not spend cycles doing validation for attackers/unauthenticated requests.